### ISSUE TITLE
Is outer_shape_bem "pitch_axis" a good name?

### ISSUE THREAD

**Issue by WillC-DNV** [IEAWindSystems/windIO#13]
Hi 

I'm looking in the WindIO ontology at the outer_shape_bem "pitch_axis" parameter. My understanding from the docs is that this parameter controls the aerofoil position relative to the reference axes, by specifying how far along the chordline the reference axes lies.

In general this location is not the pitch axis (e.g. due the the reference axis describing some prebend).

Why is it called the pitch_axis and would something like chordwise_position_of_reference_axis be more correct? 

I guess that the pitch_axis name may come from a design process where you design the geometry first before applying prebend and sweep (via the reference axis). So at this point in the design the name pitch_axis makes sense?

Thanks!

**Comment by ptrbortolotti** [comment]
Hi Will,
I agree with you, this is a good change. There was also the idea of adding a vertical offset between chord and reference axis. @fzahle will finalize soon the new blade structure for the IEA22 and that will be a good time to bring some of the changes listed here 
https://github.com/IEAWindTask37/windIO/wiki/The-next-round-of-improvements-to-windIO
Thank you for your help here,
Pietro

**Comment by jelmermc** [comment]
Agree! Perhaps adding explicitly in the name that it's relative, e.g. ‘chord_relative_stacking’ or ‘chord_relative_reference_position’?

regards, Jelmer

**Comment by fzahle** [comment]
In one of our internal geometry tools we call it “pitch axis aft leading edge” (pale). In our case, we define a reference axis, so what about reference_axis_aft_le?

**Comment by jelmermc** [comment]
Agree, that would also be clear.

**Comment by fzahle** [comment]
@ptrbortolotti in response to your ongoing PR #87.

We also discussed the need for a vertical offset of the cross-sections, so my previous proposal is perhaps not so good a choice. A vertical offset isn't hugely relevant for aero-tools, at least not BEM-based tools, but is sometimes used to improve compatibility of neighbouring sections in the inner part of the blade, which can affect structural characteristics. So perhaps going back on my own suggestion, we could also use a naming such as:

Option 1 (nomalized)

`section_offset_x`
This parameter controls the airfoil position relative to the reference axis, by specifying the distance along the chordline from the reference axis to the leading edge normalized by the local chord. 0 means that the airfoil is pinned at the leading edge, 1 at the trailing edge, 0.5 at mid-chord.

`section_offset_y`
This parameter controls the airfoil position relative to the reference axis, by specifying the chordline normal distance from the reference axis normalized by the local chord. 0 means that the reference axis lies on the airfoil chordline, a positive offset means that the chordline is shifted in the direction of the suction side relative to the reference axis, and a negative offset that the section is shifted in the direction of the pressure side of the airfoil.

Option 2 (dimensionalized):
It could also make sense to change the offsets to be dimensionalized. Two reasons: 1) while the chordwise offset makes sense as a non-dimensional quantity, the vertical offset doesn't really. 2) Controlling the leading edge curvature of the inner part of the blade is easier with a dimensioned curve.

`section_offset_x`
This parameter controls the airfoil position relative to the reference axis, by specifying the distance in meters along the chordline from the reference axis to the leading edge. 0 means that the airfoil is pinned at the leading edge, a positive offset means that the leading edge is upstream of the reference axis in local chordline coordinates, and a negative offset that the leading edge aft of the reference axis.

`section_offset_y`
This parameter controls the airfoil position relative to the reference axis, by specifying the chordline normal distance in meters from the reference axis. 0 means that the reference axis lies on the airfoil chordline, a positive offset means that the chordline is shifted in the direction of the suction side relative to the reference axis, and a negative offset that the section is shifted in the direction of the pressure side of the airfoil.

Either way a tool consuming these numbers can easily translate from dimensioned to non-dimensionalized, so it isn't hugely important to me which we choose. But it could be nice to add the vertical offset now that we're at it.

**Comment by ptrbortolotti** [comment]
I went for the dimensional option to have consistency between the two inputs

**Comment by ptrbortolotti** [comment]
PR #87 addressed this use by setting dimensional

```yaml
section_offset_x
section_offset_y
